### ISSUE TITLE
r850-alpha on Classic:  Major Bug

### ISSUE THREAD

**Issue by Amadeus-** [arkayenro/arkinventory#1209]
I recently upgraded from the last release of ArkInvenotryClassic to the r850-alpha version that was uploaded about 19 hours prior to this posting.

Anyway, a lot of things seem to work better; however, nothing new that I loot/acquire is visible in my inventory.   The slots are glowing, as usual ..but no icon.    There doesn't seem to be anything I can do to get the inventory to manually "update" or anything of that nature.  Also, /reloadui didn't seem to fix it.

Does it work for you in classic when you loot/acquire new items?

[SCREENSHOT](https://imgur.com/pydkaz8)

I'll have to downgrade back to what I was using.  Thankfully I kept a copy of my ArkInventoryClassic folders!

**Comment by arkayenro** [comment]
yep, its happily working ok in classic with new items

are there any errors?  an error might stop the draw process mid way but leave everything else working

**Comment by Amadeus-** [comment]
I'll try it again.   I had assumed it was OK to rename my settings file for ArkInventoryClassic (and the table within) so I didn't have to reconfigure everything again.  ...that's the only thing that comes to mind.

It may be tomorrow though as it's getting pretty late.  But, I'll definitely update this issue one way or another.

**Comment by arkayenro** [comment]
yes, if you copy from classic you just rename it and the database inside, or copy directly from retail

**Comment by Amadeus-** [comment]
I just logged in to test, but I can tell you that I got this error both today and yesterday:

```
4x ArkInventory\ArkInventoryUpgrades.lua:1355: bad argument #1 to 'pairs' (table expected, got nil)
[C]: ?
ArkInventory\ArkInventoryUpgrades.lua:1355: in function `DatabaseUpgradePostLoad'
ArkInventory\ArkInventory-30900.lua:2860: in function <ArkInventory\ArkInventory.lua:2831>
[C]: ?
...s\ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0-12.lua:70: in function <...s\ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:65>
...s\ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0-12.lua:527: in function `EnableAddon'
...s\ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0-12.lua:620: in function <...s\ElvUI\Libraries\Ace3\AceAddon-3.0\AceAddon-3.0.lua:605>
[C]: in function `LoadAddOn'
FrameXML\UIParent.lua:353: in function `UIParentLoadAddOn'
FrameXML\UIParent.lua:424: in function `TimeManager_LoadUI'
FrameXML\UIParent.lua:693: in function <FrameXML\UIParent.lua:658>
```

**Comment by arkayenro** [comment]
are you sure your arkinventory.lua has ARKINVDB and not something else?

line 1355 is looking for ARKINVDB.profiles so the only way for that to be missing would be for the file to exist but either completely empty, or has the wrong saved variable name in it

**Comment by Amadeus-** [comment]
This is my ArkInventory.lua file:  [LINK](https://hastebin.com/risejepodi.cs).

Everything seems to be working correctly, including my custom rules, all settings, etc.  The only problems are errors like the one I pasted above and the fact that the bug I described originally is still happening.

**Comment by Amadeus-** [comment]
I just tried copying my settings from retail, and now everything seems to work and I'm not seeing that bug.   It only seems to happen when I renamed my classic file/table.

Is there a way to clear out all of the extraneous characters stuff from the settings file after moving it from retail to classic?  I don't see anything in the configuration screens about it.

But, anyway, it appears as though my initial bug is fixed after using my retail settings file instead of the classic settings file I was using previously.   Thanks!
